modules/configuring-ovnk-additional-networks.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-additional-network.adoc
+
+:_content-type: CONCEPT
+[id="configuration-ovnk-additional-networks_{context}"]
+= Configuration for an OVN-Kubernetes additional network
+
+The {openshift-networking} OVN-Kubernetes network plugin can be configured to allow pods to have secondary network interfaces. To configure secondary network interfaces, you must define the configurations in the `NetworkAttachmentDefinition` custom resource definition (CRD).
+
+:FeatureName: Configuration for an OVN-Kubernetes additional network
+include::snippets/technology-preview.adoc[]
+
+The following sections provide example configurations for each of the topologies that OVN-Kubernetes currently allows for secondary networks.
+
+[NOTE]
+====
+Networks names must be unique. For example, creating multiple `NetworkAttachmentDefinition` CRDs with different configurations that point at the same network is unsupported.
+====
+
+[id="configuration-ovnk-network-plugin-json-object"]
+== OVN-Kubernetes network plugin JSON configuration object
+
+The following object describes the configuration parameters for the OVN-Kubernetes CNI network plugin:
+
+[cols=".^2,.^2,.^6",options="header"]
+|====
+|Field|Type|Description
+
+|`cniVersion`
+|`string`
+|The CNI specification version. The `0.3.1` value is required.
+
+|`name`
+|`string`
+|The name of the network. This value must be unique across all `NetworkAttachmentDefinitions`. Failure to make this attribute unique might result in the merging of two networks with the same name. Merging two networks by sharing similar names is unsupported.
+
+|`type`
+|`string`
+|The name of the CNI plugin to configure: `ovn-k8s-cni-overlay`.
+
+|`topology`
+|`string`
+|The topological configuration for the network. The value must be `layer2`.
```

Comment:
So here and in `cniVersion` and `type` above, I think you are specifying required values. Consider using parallel phrasing for these three items to make it clear that they are all prescribed values.